content/en/network_monitoring/network_path/setup.md

```diff
@@ -34,6 +34,8 @@ Setting up Network Path involves configuring your Linux environment to monitor a
 {{< tabs >}}
 {{% tab "Dynamic paths" %}}
 
+Configure dynamic paths to allow the Agent to automatically discover and monitor network routes without requiring you to specify endpoints manually.
```

Comment:
```suggestion
Configure dynamic paths to allow the Agent to automatically discover and monitor network routes based on actual network traffic without requiring you to specify endpoints manually.
```

I'm wondering if it can help to add "based on actual network traffic" to let user know how those dynamic paths are created.